searxng/searxng
# theme: fix "#backToTop" button always being clickable

## What does this PR do?


The button to scroll back to the top is always clickable, even if you've not scrolled down, it'll just have it's opacity set to `0`. This means hovering over where the button is, will cause the cursor/pointer to become that of the `pointer` style, yet nothing will happen when clicking, as you're already at the top.


## Why is this change important?


I can imagine that a button having pointer events when it does nothing could trigger some problems in terms of accessibility clients. Further there's no good reason why it should be able to clicked or hovered (from what I know).


## How to test this PR locally?



 1. Start the dev server (`make run`)
 2. Make a search query
 3. Scroll slightly
 4. Hover over the "back to top" button
 5. Scroll back to top, whilst keeping the mouse still
 6. The button is no longer clickable, and the cursor changes accordingly

Do the same before this PR, and it'll then still be clickable and hoverable, and have it's cursor indicate it's clickable.
